commit f25b2fb76de63b44400412edbbc4e668c3dc6c50 (refs/changes/99/899/1)
Author: David Pacheco <dap@joyent.com>
Date:   2016-11-14T15:24:16-08:00 (2 years, 11 months ago)
    
    TOOLS-1613 cr.joyent.us needs to work around gerrit issue 3885

diff --git a/images/appserver/etc/gerrit.config.base b/images/appserver/etc/gerrit.config.base
index ef22120..79b5d2f 100644
--- a/images/appserver/etc/gerrit.config.base
+++ b/images/appserver/etc/gerrit.config.base
@@ -16,6 +16,7 @@
 
 [index]
 	type = LUCENE
+	threads = 1
 
 [auth]
 	type = OAUTH
